File: LuaLibs/src/MyLua/Libs/DelayHandle/DelayHandleObject.lua
require "MyLua.Libs.Core.GlobalNS"

local M = GlobalNS.Class()
M.clsName = "DelayHandleObject"
GlobalNS[M.clsName] = M

function M:ctor()
    self.m_delayObject = nil;
    self.m_delayParam = nil;
end

return M

File: PyLibs/src/Libs/DelayHandle/DelayHandleObject.py
# -*- coding: utf-8 -*-

from Libs.Core.GObject import GObject;

class DelayHandleObject(GObject):
    def __init__(self):
        self.mTypeId = "DelayHandleObject";
        self.mDelayObject = None;
        self.mDelayParam = None;

